### DISCUSSION TITLE
please help with my video not loading problem

### DISCUSSION THREAD

**Discussion by kenankurta** [community/community#48223] [Other Feature Feedback, Questions, & Ideas] [answered]
so i had a previous problem with images not loading but now i have fixed id. well now everything is loading except for one video
anyone know why is that?
[InformatikaProjekat2.github.io](https://github.com/kenankurta/InformatikaProjekat2.github.io)

**Comment by radar-blip** [comment]
Have you fixed this?
One of your mp4's has spaces in the filename. That could be causing the problem.

**Reply by kenankurta** [reply]
nope and i tried i think every way to write it and what other people have used but it still doesnt work

**Comment by melvincwng** [comment, marked as answer]
@kenankurta, the issue is with line 91 of project1.html over [here](https://github.com/kenankurta/InformatikaProjekat2.github.io/blob/main/project1.html#L91). Essentially that line 91 is invalid HTML syntax.

Try replacing line 91 with this particular block of html code 
`<source src="./css/videovatre.mp4" type="video/mp4">`.

Replacing your video source path in your HTML code should resolve this issue of the video not playing.

Hope this clarifies and answers your question 😃

**Reply by kenankurta** [reply]
yeah it helped thanks

**Reply by tioluwanimofe** [reply]
Thanks for this

**Comment by Anupmqr** [comment]
Reduce video file size
